.github/workflows/deploy.yml

```
@@ -24,127 +26,100 @@ jobs:
-      - name: Download build artifacts
-        uses: actions/download-artifact@v4
-        with:
-          name: lambda-jar
-          path: build/libs/
+      - name: Write params-dev.json from Secrets
+        run: printf '%s' '${{ secrets.PARAMS_DEV_JSON }}' > params-dev.json
 
-      - name: Set up Node.js
-        uses: actions/setup-node@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.9)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)